scripts.js

```diff
@@ -0,0 +1,226 @@
+  }
+
+  makeRequest() {
+    fetch(this.requestLink)
+      .then(res => res.json())
+      .then(json => {
+        let result = [...json.results];
+        let quizCategory = result[0].category;
+        result.forEach(obj => {
+          this.correctRequestAnswers.push(obj.correct_answer);
+          this.questionRequest.push(obj.question);
+          this.allAnswersRequest.push(...obj.incorrect_answers, obj.correct_answer);
+        });
+
+        document.querySelector('.title').textContent = `Quiz about ${quizCategory}`;
+
+        this.insertText(this.questionRequest, this.allAnswersRequest);
+        json.onload = this.preloader.classList.add('none');
+      })
+      .catch(err => console.log(err));
+  }
+
+  insertText(arrQuestion, arrAnswer) {
+    const questionText = this.container.querySelectorAll('.question__title');
+    for (let i = 0; i < arrQuestion.length; i++) {
+      questionText[i].textContent = arrQuestion[i];
+    }
+
+    const answerText = this.container.querySelectorAll('.answer');
+    const valueText = this.container.querySelectorAll('.radio');
+    for (let i = 0; i < arrAnswer.length; i++) {
+      answerText[i].textContent = arrAnswer[i];
+      valueText[i].value = arrAnswer[i];
+    }
+  }
+
+  assignHandlerAnswer() {
+    Array.from(this.container.querySelectorAll('.question .question__answers'))
+      .forEach(answers => {
+        answers.addEventListener("click", this.checkResultBind);
+      })
+  }
+
+  assignHandlerButton() {
+    this.start.addEventListener("click", this.startGame.bind(this));
+    this.next.addEventListener("click", this.switchNextQuestion.bind(this));
+    this.previous.addEventListener("click", this.switchPreviousQuestion.bind(this));
+    this.buttonName.addEventListener("click", this.eventButtonName.bind(this));
+    this.restart.addEventListener("click", this.eventRestart.bind(this));
+  }
+
+  checkResult(event) {
+    let target = event.target;
+    if (target.tagName != 'LABEL') return;
+
+    this.disableButton();
+
+    let answer = target.previousElementSibling.value;
+    let isCorrect = this.correctDefaultAnswers.includes(answer) || this.correctRequestAnswers.includes(answer);
+    if (isCorrect) {
+      target.style.background = "green";
+      this.countAnswers++;
+    } else {
+      target.style.background = "red";
+    }
+
+    clearTimeout(this.timerShowQuestion);
+
+    this.timerCheckQuestion = setTimeout(this.switchNextQuestion.bind(this), 1500);
+  }
+
+  disableButton() {
+    const disableBtn = this.slides[this.countSlide].querySelectorAll('.radio');
+    Array.from(disableBtn).forEach(item => item.setAttribute("disabled", "disabled"));
+
+    const disableAnswers = this.slides[this.countSlide].querySelector('.question__answers');
+    disableAnswers.removeEventListener("click", this.checkResultBind);
+  }
+
+  startGame() {
+    this.start.classList.add('none');
+    this.next.classList.remove('none');
+    this.previous.classList.remove('none');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/game-js/pullRequest?prid=7705696)
```suggestion
    this.previous.classList.remove("none");
```